### PR TITLE
Run konflux builds and tests on all PRs

### PR DESCRIPTION
**Pull request by Molter73**
## Description

This should help us prevent breaking this build on master.

## Checklist
- [x] Investigated and inspected CI test results
- [ ] Updated documentation accordingly

**Automated testing**
  - [ ] Added unit tests
  - [ ] Added integration tests
  - [ ] Added regression tests

If any of these don't apply, please comment below.

## Testing Performed

- [x] Check all konflux jobs run in the PR.

### CODE REVIEW

File: .github/workflows/konflux.yml [hunk 1]
```diff
@@ -20,10 +20,85 @@ concurrency:
   cancel-in-progress: true
```
Comment (msugakov) [inline .github/workflows/konflux.yml:21]: [two things, not about this line]

One:
```yaml
name: Run konflux workflows
```
may suggest that GitHub drives/triggers Konflux which is not the case. I'd propose to change the name along the lines of "Run post-Konflux workflows" or "Test Konflux builds" or something.

Two:
This has cron schedule, I wonder what's the use?

**Review by msugakov — APPROVED** [review]
The comment/s is/are not blocking.

File: .github/workflows/konflux.yml (hunk 1 above)
Comment (Molter73) [reply .github/workflows/konflux.yml:21]: > One:
> 
> ```yaml
> name: Run konflux workflows
> ```

I agree, I'll rename the workflow.

> Two:
> This has cron schedule, I wonder what's the use?

I believe this was added by @Stringy in order to run the konflux tests on nightlies, but I don't think it has much point if we don't trigger the konflux build as well. I'll ask him about it.

**Review by msugakov — APPROVED** [review]

**Review by Stringy — APPROVED** [review]
LGTM!

**Pull request merged**